Chapters/6_Qualified_identifiers.md

```diff
@@ -86,18 +86,34 @@ indicates that the content seen previously with the function `generate_intput_st
 been seen in the Git repository at `https://gitorious.org/ocamlp3l/ocamlp3l_cvs.git`, when
 its full state had the SWHID core identifier `swh:1:snp:d7f1b9eb7ccb596c2622c4780febaa02549830f9`, and that it is named `simplefarm.ml` in the directory `Simplefarm` contained in the directory `Examples` contained in the root directory associated to the revision with core SWHID `swh:1:rev:2db189928c94d62a3b4757b3eec68f0a4d4113f0`.
 
+### 6.3 Comparing qualified SWHIDs
 
-### 6.3 Recommendations
+One can determine whether two software artifacts are identical (bit by bit) by
+comparing their core SWHIDs, ignoring all qualifiers. If the core SWHIDs match,
+the software artifacts they represent are identical.
+
+To determine if two SWHIDs represent the same software artifact (or fragment
+thereof) in the same context, one must also compare their qualifiers. Two SWHIDs
+are considered equivalent in context if:
+
+- They both have the same set of qualifiers.
+- The values of these qualifiers are identical. For instance, if both SWHIDs
+  have an `anchor` qualifier, the core SWHID values of these qualifiers should
+  match. Similarly, if both have a `lines` qualifier, their values should be the
```

Comment:
Minor as above + the fact "should" here is incorrect: "value of these qualifiers should match" -> "values of these qualifier are identical".